### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by ghost**
added a script to scrape list of hackathons from hackerearth.com/hackathons
the scraped data is dumped into individual JSON files

### CODE REVIEW

**Comment by ghost** [comment]
@ujjwal96 kindly review the code once and let me know of any changes
@UjjwalAyyangar here is the code you requested to see

**Comment by ashwini0529** [comment]
@gargraina03 Do it in a single file may be? 
Further implementation will be to add it to db. What do you suggest @ujjwal96 ?

**Comment by ghost** [comment]
@ashwini0529 sure i'll make a single file of all the details

File: script1.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+path = r"./chromedriver"
+wd = webdriver.Chrome(path)
+wd.get("https://www.hackerearth.com/hackathons/")
+
+html = wd.page_source
+html = html[html.index("""<script type="application/ld+json">"""):]
+html = html[:html.index("</script>")]
+
+soup = bS(html, 'lxml')
```
Comment (r00tdaemon) [inline script1.py:16]: Use html5lib, that's what we are using in other scrapers

**Comment by r00tdaemon** [comment]
There are few things that are need to be done.
- Wrap everything inside a function. 
- Get rid of the dumps. We don't need to save the dumps. Just return them from the function.
- Instead of making a separate file add this function to the existing one.

if you have any queries ask on slack.

**Pull request closed without merging**